packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -371,10 +372,19 @@ export class SummarizerNode implements IRootSummarizerNode {
 					return { latestSummaryUpdated: false };
 				}
 
+				let latestSnapshotFetch: IFetchSnapshotResult;
```

Comment:
This does not look right. It's more of a workaround. There are too many try-catch blocks here + container runtime which are trying to do similar things. A try-catch block around the outermost call in container runtime (this.summarizerNode.refreshLatestSummary) should be sufficient, no?